h2/src/main/org/h2/mvstore/MVStore.java

```diff
@@ -664,8 +664,8 @@ private synchronized void readStoreHeader() {
         loadChunkMeta();
         // read all chunk headers and footers within the retention time,
         // to detect unwritten data after a power failure
-        verifyLastChunks();
```

Comment:
You moved this call to the end (why?), but didn't move the comment. Could you do that as well?

Reply:
When filestore is re-used between MVStore sessions location of fileStore.clear(); call become important which was revealed by latest assumptions I put in. A couple of tests start failing.
verifyLastChunk() call rollbackTo() which does chunks manipulations and FreeSpaceBitSet of the store should be intact by then.